src/Core/InfoBar/IInfoBarManager.cs

```diff
@@ -54,6 +54,35 @@ public interface IInfoBarManager
         /// </remarks>
```

Comment:
Technically we could merge all three methods into one and replace the buttonTexts with params, the only reason I thought about that is because now this class has 6 methods which are almost identical. Changing it would require touching quite a few classes though so I didn't bother.